src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -10,5 +10,5 @@
       "type": "java.lang.String",
       "description": "Some salt to be added to the initial vector."
     }
-  ]}
-  
\ No newline at end of file
+  ]
+}
```

Comment:
Is there a newline character at the end of this line?

Reply:
yes